src/map/utils/battleutils.cpp

```diff
@@ -5595,8 +5595,14 @@ namespace battleutils
         }
     }
 
-    bool DrawIn(CBattleEntity* PTarget, CMobEntity* PMob, float offset)
+    bool DrawIn(CBattleEntity* PTarget, CMobEntity* PMob, float offset, uint8 drawInRange)
     {
+        // Check if time to draw in again, if not then exit
+        if (std::chrono::time_point_cast<std::chrono::seconds>(server_clock::now()).time_since_epoch().count() - PMob->GetLocalVar("DrawInTime") < 2)
```

Comment:
I'm fairly sure we have defines that allow you to make `std::chrono::time_point_cast<std::chrono::seconds>(server_clock::now()).time_since_epoch().count()` much shorter than this? Or you can define it on the line before as const auto and then use it in the if statement

Reply:
I wasn't able to find a define like this for seconds, so I made a variable and called it like you mentioned.